x-pack/plugins/security_solution/public/detections/pages/detection_engine/detection_engine.tsx

```diff
@@ -291,15 +291,7 @@ const DetectionEnginePageComponent: React.FC<DetectionEngineComponentProps> = ({
   }, []);
 
   // Callback for when open/closed filter changes
-  const onFilterGroupChangedCallback = useCallback(
-    (newFilterGroup: Status) => {
-      const timelineId = TableId.alertsOnAlertsPage;
-      clearEventsLoading({ id: timelineId });
-      clearEventsDeleted({ id: timelineId });
-      setStatusFilter([newFilterGroup]);
-    },
-    [clearEventsLoading, clearEventsDeleted, setStatusFilter]
-  );
+  const onFilterGroupChangedCallback = useCallback((newFilterGroup: Status) => {}, []);
```

Comment:
This isn't doing anything anymore? Do we not need to clear the loading/deleted states or update the status filter onChange anymore?

Reply:
+1, wondering this as well.

Reply:
Sorry, this change should have been in another branch. I have reverted this. In a separate branch, I was trying to remove old Alert State toggle from the alert page so that feature flag of Alert Page Controls will no longer be needed.

But now I am not sure, because we need to keep it in the minor releases of 8.7 so for now I will keep it and we can remove it later. Hence, reverted that change to same as `main`